### PR TITLE
Recommended scaffolding for spatial packages

### PR DESCRIPTION
**Pull request by mpadge**
In response to these [two](https://github.com/ropensci/software-review-meta/issues/47#issuecomment-973245903) [comments](https://github.com/ropensci/software-review-meta/issues/47#issuecomment-973905554) in https://github.com/ropensci/software-review-meta/issues/47

### CODE REVIEW

File: pkg_building.Rmd [hunk 1]
```diff
@@ -399,6 +399,8 @@ rather than one where it is re-exported. For instance many functions in **devtoo
 
 * For parsing, creating, and manipulating XML, we strongly recommend [xml2][] for most cases. [You can refer to Daniel Nüst's notes about migration from XML to xml2](https://gist.github.com/nuest/3ed3b0057713eb4f4d75d11bb62f2d66).
 
+* For spatial data, the [sp][] package should be considered deprecated in favour of [sf][], and the packages [rgdal][], [maptools][], and [rgeos][] will be retired by the end of 2023. We recommend use of the spatial suites developed by the [r-spatial](https://github.com/r-spatial) and [rspatial](https://github.com/rspatial) communities.
```
Comment (maelle) [inline pkg_building.Rmd:402]: Thanks! Would you mind adding a link to the software-review-meta thread that we can hopefully replace with some sort of blog post link at one point in the future

**Review by maelle — CHANGES_REQUESTED** [review]
Thanks a lot, so cool to see a clear recommendation! Could you please also update the changelog?

File: pkg_building.Rmd (hunk 1 above)
Comment (mpadge) [reply pkg_building.Rmd]: done, thanks!

**Review by maelle — APPROVED** [review]
Thanks a ton!

**Comment by mpadge** [comment]
Always a pleasure!! :smile:

**Pull request merged**